sdk/tests/client/input_selection/basic_outputs.rs

```diff
@@ -1582,14 +1582,14 @@ fn two_inputs_remainder_3() {
     .select()
     .unwrap();
 
-    assert!(unsorted_eq(&selected.inputs_data, &inputs));
+    assert!(unsorted_eq(&selected.inputs_data, &inputs[1..]));
```

Comment:
The name of the test should be changed or you change the amounts so two inputs are needed

Reply:
done